docs/validations.stories.mdx

```diff
@@ -27,52 +29,39 @@ Each input component which supports validations accepts following props - `error
 
 Passing a `string` to these props will display a border along with a validation icon and tooltip; this `string` value will be displayed as the tooltip message.
 
-```js
-<Textbox name="name" value={value} onChange={onChange} error="Message" />
-```
+<Canvas>
+  <Story name="string validation">
+    <Textbox name="name" value="value" error="Message" />
+  </Story>
+</Canvas>
 
 Passing a `boolean` to these props will display a border and validation icon without the additional tooltip information.
 
-```js
-<Textbox name="name" value={value} onChange={onChange} error={true} />
-```
+<Canvas>
+  <Story name="boolean validation">
+    <Textbox name="name" value="value" error={true} />
+  </Story>
+</Canvas>
 
 The only exception from this is the `DateRange` component which displays separate validation statuses on either `start` or `end` dates - to achieve this the `DateRange` component accepts `startError`, `endError`, `startWarning`, `endWarning`, `startInfo` and `endInfo` props.
 
-```js
-<DateRange
-  name="name"
-  value={value}
-  onChange={onChange}
-  startError="Start message"
-  endError="End message"
-/>
-```
+<Canvas>
+  <Story name="date range validation">
+    <DateRange
+      name="name"
+      value={["01/10/2016", "30/10/2016"]}
+      startError="Start message"
+      endError="End message"
+    />
```

Comment:
```suggestion
    <DateRange
      name="name"
      value={["01/10/2016", "30/10/2016"]}
      onChange={() => {}}
      startError="Start message"
      endError="End message"
    />
```
This is a minor issue, we do get a console error because `onChange` is a required prop for `DateRange`.